packages/fx-core/src/plugins/resource/bot/index.ts

```diff
@@ -203,9 +204,13 @@ export class TeamsBot implements Plugin {
         telemetryHelper.sendResultEvent(
           context,
           name,
-          ResultFactory.SystemError(UnhandledErrorCode, `Got an unhandled error: ${e.message}`)
+          ResultFactory.SystemError(
+            UnhandledErrorCode,
+            `Got an unhandled error: ${e.message}`,
+            e.innerError
+          )
         );
-      return ResultFactory.SystemError(UnhandledErrorCode, e.message, e);
+      return ResultFactory.SystemError(UnhandledErrorCode, e.message, e.innerError);
```

Comment:
I'm wondering if the main purpose is to remove message of inner error from e.message?

Reply:
yes, for in api its recent change will joint the inner error msg